lib/containers/runtime.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test Docker’s btrfs storage driver features for image and container management
+# Maintainer: qac team <qa-c@suse.de>
+package containers::runtime;
+use Mojo::Base qw(consoletest);
+use testapi;
+use Test::Assert ':all';
+
+sub new{
+    my ($class, $runtime) = @_;
+    my $self = {
+	runtime => $runtime,
+    };
+    bless $self, $class;
+    return $self;
+}
+
+sub build {
+    my ($self, $dockerfile_path, $container_tag) = @_;
+    die 'wrong number of arguments' if @_ < 3;
+    assert_script_run "cd $dockerfile_path";
+    assert_script_run "$self->{runtime} build -t $container_tag .";
```

Comment:
what about wrapping `assert_script_run()`, `script_run()`, `script_output()` a.s.o, so you don't need to write this `$self->{runtime}` over and over.

Reply:
Done.